WalletWasabi/JsonConverters/EndPointJsonConverter.cs

```diff
@@ -44,11 +38,11 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 		}
 
 		/// <inheritdoc />
-		public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+		public override void WriteJson(JsonWriter writer, EndPoint? value, JsonSerializer serializer)
 		{
-			if (value is EndPoint endPoint)
+			if (value is { })
```

Comment:
Should we just not do anything in the else part?

Reply:
```suggestion
			if (value is not null)
```
is more common these days.

Reply:
(When you write JSON, I would expect that you serialize C# null as JSON null. That would be common general behavior.)